user-guide/python.rst

```diff
@@ -1,98 +1,34 @@
 Using Python
 ============
 
-Python on Cirrus is provided by the `Anaconda <https://www.continuum.io/>`__
-distribution. The Python 3 version of the distribution is available.
+Python on Cirrus is provided by a number of `Miniconda <https://conda.io/miniconda.html>`__ modules and one `Anaconda <https://www.continuum.io>`__ module.
+(Miniconda being a small bootstrap version of Anaconda).
 
-The central installation provides many of the most common packages used for
-scientific computation and data analysis.
+The Anaconda module is called ``anaconda/python3`` and is suitable for running serial applications.
 
-If the packages you require are not included in the central Anaconda Python
-distribution, then the simplest way to make these available is often to install
-your own version of `Miniconda <https://conda.io/miniconda.html>`__  and add the
-packages you need. We provide  instructions on how to do this below.
+You can list the Miniconda modules by running ``module avail python`` on a login node. Those module versions that have the ``gpu`` suffix are
+suitable for use on the `Cirrus GPU nodes <https://cirrus.readthedocs.io/en/main/user-guide/gpu.html>`__. There are also modules that extend these Python environments, e.g., ``pyfr``, ``horovod``,
```

Comment:
This link should probably be converted to an internal/relative link

Reply:
Done